conf/app.routes

```diff
@@ -21,3 +21,6 @@ GET         /unauthorised                    controllers.errors.UnauthorisedCont
 
 GET         /disclosure/entry-details        controllers.EntryDetailsController.onLoad
 POST        /disclosure/entry-details        controllers.EntryDetailsController.onSubmit
+
+GET         /disclaimer            controllers.SupportingDocController.onLoad
```

Comment:
The route is incorrect, could you check with the content spreadsheet linked in the Jira please.